CMakeLists.txt

```diff
@@ -736,27 +738,10 @@ if (NOT LWS_WITHOUT_TESTAPPS)
 			set(TEST_SERVER_SSL_CERT "${PROJECT_BINARY_DIR}/libwebsockets-test-server.pem")
 
 			if (WIN32)
-				file(WRITE "${PROJECT_BINARY_DIR}/openssl_input.txt"
```

Comment:
Don't remove the normal windows support. Instead you use use `if (MINGW)` instead, if its different from normal Win32